lib/koans/15_processes.ex

```diff
@@ -18,7 +18,9 @@ defmodule Processes do
   end
 
   koan "New processes are spawned functions" do
-    pid = spawn(fn -> nil end)
+    pid = spawn(fn -> receive do
+                      end
+                end)
```

Comment:
Thanks for going ahead with at least fixing this problem! (Despite the examples themselves may needed to be clearer). Do you have any specific opinions about the style of this? The way it is right now is a little 😖 . What do you think about naming the function?

```elixir
wait_forever = fn ->
  receive do
  end
end
pid = spawn(wait_forever)
```

Maybe that would help make things a little clearer in the mean time? I had also used the form `receive(do: nil)` in #166, but I'm not sure how confusing that is.